### DISCUSSION TITLE
How to create flux-system secret after rebootsrapping?

### DISCUSSION THREAD

**Discussion by preston-willis** [fluxcd/flux2#4220] [General]
I rebootstrapped flux  and deleted the flux-system secret (bootstrapping with bitbucket over https). I am unable to regenerate the flux-system secret and my helm repositories are failing because of this. Has anyone had a similar issue?

**Comment by makkes** [comment]
> I am unable to regenerate the flux-system secret

Why? What is the issue specifically?

You can always bootstrap again. Flux will notice the missing Secret and generate a new one.
